cpp/nodes_module/algorithm/nodes.cpp

```diff
@@ -1 +1,59 @@
-#include "nodes.hpp"
\ No newline at end of file
+#include "nodes.hpp"
+
+bool Nodes::RelationshipExist(const mgp::Node &node, std::string &rel_type) {
+  char direction{' '};
+  if (rel_type[0] == '<' && rel_type[rel_type.size() - 1] == '>') {
+    throw mgp::ValueException("Invalid relationship specification!");
+  }else if (rel_type[rel_type.size() - 1] == '>') {
+    direction = rel_type[rel_type.size() - 1];
+    rel_type.pop_back();
+  }else if (rel_type[0] == '<') {
+    direction = rel_type[0];
+    rel_type.erase(0,1);
+  }
+  for (auto rel : node.OutRelationships()) {
+    if (std::string(rel.Type()) == rel_type && direction != '<') {
+      return true;
+    }
+  }
+  for (auto rel : node.InRelationships()) {
```

Comment:
const auto &rel

Reply:
I think we don't need ref as Relationship object is returned as Value, there isn't reference to something that exists.